swo_aws_extension/flows/jobs/billing_journal.py

```diff
@@ -846,3 +878,25 @@ def _is_service_discount_valid(self, amount, discount, target_discount):
         if abs(discount - target_discount) > self.config.billing_discount_tolerance_rate:
             return False
         return True
+
+    @staticmethod
+    def _get_support_discount(support_metrics, refund_metrics):
+        """
+        Calculates the support discount based on the refund and support amounts from the
+        per record cost metric.
+        Args:
+            support_metrics (dict): Support metrics from the AWS report.
+            refund_metrics (dict): Refund metrics from the AWS report.
+        Returns:
+            int: Support discount percentage rounded to the nearest integer.
+        """
+        support = 0
+        refund = 0
+
+        for _, amount in support_metrics.items():
+            support = amount
```

Comment:
Is it suppose to be `+=`?

Reply:
In this case, we can use -> 

`sum(support_metrics.values())`

Same [here](https://github.com/softwareone-platform/swo-aws-extension/pull/103/files#diff-d729bd0b1da39689d748a926c3065765b720e6f560626d176a8e237efa938b54R899)  -> `sum(refund_metrics.values())`